models/DataObject/ClassDefinition/Service.php

```diff
@@ -16,11 +16,12 @@
 
 namespace Pimcore\Model\DataObject\ClassDefinition;
 
-use Pimcore\Loader\ImplementationLoader\LoaderInterface;
-use Pimcore\Logger;
 use Pimcore\Model\DataObject;
 use Pimcore\Model\DataObject\ClassDefinition\Data\EncryptedField;
+use Pimcore\Logger;
+use Pimcore\Loader\ImplementationLoader\LoaderInterface;
 use Pimcore\Tool;
+use Pimcore\Model\DataObject\ClassDefinition\Data\VarExporterInterface;
```

Comment:
This is the correct order. The whole namespace should be alphabetic, not only the class name.
```suggestion
use Pimcore\Loader\ImplementationLoader\LoaderInterface;
use Pimcore\Logger;
use Pimcore\Model\DataObject;
use Pimcore\Model\DataObject\ClassDefinition\Data\EncryptedField;
use Pimcore\Model\DataObject\ClassDefinition\Data\VarExporterInterface;
use Pimcore\Tool;
```

Reply:
thanks 👍